index.html

```diff
@@ -174,6 +174,13 @@ <h5 class="docList-header"><code>options.characters</code></h5>
                         </div>
                     </li>
                     <li class="docList-item">
+                        <h5 class="docList-header"><code>options.exclude</code></h5>
+                        <div class="docList-info">
+                            <p>These are the characters that baffle ignores in your text when obfuscating it. You can pass in an array of strings.</p>
```

Comment:
Let's change the language here so it reads "...an array of characters." I know it's technically an array of strings, but I think that communicates the intention more clearly.